aeson.cabal

```diff
@@ -174,6 +174,9 @@ test-suite tests
   if !impl(ghc >= 7.10)
     build-depends: nats >=1 && <1.2
 
+  if impl(ghc >= 7.8)
+    build-depends: hashable-time >= 0.2 && <0.3
+
```

Comment:
What does this mean for older GHC's, and why is it only needed in the test-suite?